Kitodo/src/main/java/org/kitodo/production/forms/dataeditor/DataEditorForm.java

```diff
@@ -324,6 +324,7 @@ private void init() {
         structurePanel.getSelectedPhysicalNode().setSelected(true);
         metadataPanel.showLogical(getSelectedStructure());
         metadataPanel.showPhysical(getSelectedPhysicalDivision());
+        galleryPanel.setGalleryViewMode(GalleryViewMode.getByMessageKey(user.getDefaultGalleryViewMode()).name());
```

Comment:
In my opinion, we should not use message keys for behavioral testing in logic, because it can change and does not describe a state. I think here user.getDefaultGalleryViewMode() should directly return a GalleryViewMode or only the name of the enum should be used. Conceivable would be here also a number. 0 for the list 1 for the preview etc.

Reply:
Good point. Implemented.